internal/cmd/config/config_test.go

```diff
@@ -524,3 +524,58 @@ func TestController_EventingConfig(t *testing.T) {
 		})
 	}
 }
+
+func TestParsingPath(t *testing.T) {
```

Comment:
Would be good to create a temp file and set a `os.SetEnv` to fully test the happy path

Reply:
I thought about this, but I didn't end up going that path since that functionality should be the responsibility of `parseutil.ParsePath` to get right, so I was divided as to whether I should've included that in these tests or not.

I can certainly add it though, not much more work really!